.docksal/etc/conf/settings.kanopi.php

```diff
@@ -18,6 +29,9 @@
 $settings['file_chmod_directory'] = 0777;
 $settings['file_chmod_file'] = 0666;
 
+//Enable access to rebuild.php.
```

Comment:
Nit: please add a space `// Enable access to rebuild.php.`

Reply:
fixed